KinWallet/View Controllers and Co./More/KinSupportViewController.swift

```diff
@@ -131,7 +131,7 @@ final class KinSupportViewController: MFMailComposeViewController {
         mailController.addAttachmentData(attachmentData(),
                                          mimeType: "txt",
                                          fileName: "Info.txt")
-        mailController.mailComposeDelegate = mailController
+      //  mailController.mailComposeDelegate = mailController
```

Comment:
Make sure to remove this comment. Also, you can remove the [import at line 7](https://github.com/kinecosystem/kinit-ios/blob/e1a1116338423632dc408a51dbb1cedf06b6f38d/KinWallet/View%20Controllers%20and%20Co./More/KinSupportViewController.swift#L7)